### PR TITLE
Fix 3dCompSurfelData

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Fix   --noWindows options see issue #427 
Thanks @copyme !

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).
- [x] All continuous integration tests pass (Github Actions & appveyor).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -22,7 +22,9 @@
     (David Coeurjolly [#413](https://github.com/DGtal-team/DGtalTools/pull/413))
   - 3dVolViewer: adding new option to interprate intensity as transparency.
     (Bertrand Kerautret [#425](https://github.com/DGtal-team/DGtalTools/pull/425))
-
+  - 3dCompSurfelData: fix --noWindows options (issue [#427](https://github.com/DGtal-team/DGtalTools/issues/427).
```
Comment (dcoeurjo) [inline ChangeLog.md:25]:
```suggestion
  - 3dCompSurfelData: fix --noWindows option (issue [#427](https://github.com/DGtal-team/DGtalTools/issues/427). 
```

**Review by dcoeurjo — CHANGES_REQUESTED** [review]
👌 just minor stuff

**Comment by kerautret** [comment]
thanks @dcoeurjo  done ;)

**Review by dcoeurjo — APPROVED** [review]
👍

**Pull request merged**